crates/matrix-sdk/src/widget_api/messages/message.rs

```diff
@@ -1,27 +1,45 @@
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Serialize, Deserialize)]
-pub struct Message<Req, Resp> {
+use super::{FromWidgetAction, ToWidgetAction};
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "api")]
+pub enum Message {
+    #[serde(rename = "fromWidget")]
+    FromWidget(FromWidgetAction),
+    #[serde(rename = "toWidget")]
+    ToWidget(ToWidgetAction),
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct ActionBody<Req, Resp> {
```

Comment:
It feels to me that the usage of `Action` might be confusing given that many messages are not actions (i.e. `ContentLoaded` does not seem like an action). It feels like they are just called `action` for historical reasons in the MSC, but logically they are just messages. Should we opt for usage of a `Message` name maybe or something similar?